tests/SimpleStringTest.cpp

```diff
@@ -560,20 +560,15 @@ TEST(SimpleString, CollectionWritingToEmptyString)
     STRCMP_EQUAL("", col[3].asCharString());
 }
 
-#ifdef CPPUTEST_64BIT
-#ifdef CPPUTEST_64BIT_32BIT_LONGS
+#if !defined(CPPUTEST_64BIT) || \
+     defined(CPPUTEST_64BIT_32BIT_LONGS)
```

Comment:
That's the idea.

I didn't like to different #else block for essentially the same thing,